keyboards/tkw/stoutgat/v2/keymaps/default/keymap.c

```diff
@@ -0,0 +1,79 @@
+#include QMK_KEYBOARD_H
+
+enum layers {
+    _BL,
+    _FL,
+};
+
+
+
+   const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+     [0] = LAYOUT_encoder(
+       KC_MPLY,KC_ESC,
```

Comment:
Correct me if I'm wrong, but:

- This keymap is copied and modified from the V1 PCB.
- Looking at the schematics from your GitHub, `KC_ESC` here seems like it wouldn't actually work:
  - The macro assigns this position to `K48`, which exists on the V1 but not on the V2.
  - The V2 seems to support three rotary encoders:
    - `SW18`: Ctrl + Right or Left on the Base layer when turned, in the place that is normally the Escape key. `KC_GESC` when tapped.
    - `SW81`: Also Ctrl + Right or Left on the Base layer when turned (same encoder pins as `SW18`), located next to the left Shift key. `KC_MPLY` when tapped.
    - `SW12`: Page Down or Up when turned, `KC_GRV` when tapped. Located in the top right corner of the board.

Reply:
I've removed the K48 from the macro, thanks for spotting that.
The idea is that you either mount an encoder at SW18 OR SW81, but not both.